### PR TITLE
Update  Image and APT Mirror docs (Bielefeld cloud site)

### PR DESCRIPTION
**Pull request by jkrue**
Try to fulfill the following points before the Pull Request is merged:
- [x] Please create your prs against the dev branch
- [x] Please WAIT for the tests to finish.
- [ ] All tests have been successfully completed.
- [x] When adding a new documentation or tutorial, make sure that the markdown is linked in the config.yml

### CODE REVIEW

**Comment by jkrue** [comment]
Create PR against master since the development branch seems to be outdated.

**Comment by jkrue** [comment]
HTML Link checker seems to be broken. Got a timeout for a valid URL.

File: wiki/Compute_Center/Bielefeld.md [hunk 1]
```diff
@@ -64,17 +64,17 @@ consider this if running docker or any other container technology.
 
 ## Images
 
-We provide some preconfigured cloud images on top of the Ubuntu LTS (18.04 and 20.04) and Debian (10). These 
-images are able to run on other cloud sites without any further modifications and come with a 
-script `/usr/local/bin/de.NBI_Bielefeld_environment.sh` that adapts a running instance to 
-the cloud site Bielefeld:
+We provide some preconfigured cloud images on top of Ubuntu LTS (18.04, 20.04, and 22.04) and Debian (10 and 11). 
+These images are preconfigured to adapt a running instance to the cloud site Bielefeld:
```
Comment (maitai) [inline wiki/Compute_Center/Bielefeld.md:68]:
```suggestion
Preconfigured cloud images are available for Ubuntu LTS (18.04, 20.04, and 22.04) and Debian (10 and 11). 
These images apply some auto-configuration on boot specific to the cloud site Bielefeld:
```

File: wiki/Compute_Center/Bielefeld.md [hunk 2]
```diff
@@ -64,17 +64,17 @@ consider this if running docker or any other container technology.
...
 
-- set proxy for environment, apt and docker if necessary
-- make use of apt-mirror
+- make use of a local apt-mirror by default
+- set proxy for the environment, apt and docker if necessary
+
+All images are able to run on other cloud sites without any further modifications.
```
Comment (maitai) [inline wiki/Compute_Center/Bielefeld.md:73]:
```suggestion
- Site-local APT mirror is used as default
- If needed, APT, Docker and the environment are configured to use the local proxy

All images are able to run on other cloud sites without any further modifications.
```

**Review by maitai — APPROVED** [review]

**Pull request merged**